apps/laboratory/src/pages/library/wagmi.tsx

```diff
@@ -25,7 +25,6 @@ const modal = createWeb3Modal({
   metadata: ConstantsUtil.Metadata,
   termsConditionsUrl: 'https://walletconnect.com/terms',
   privacyPolicyUrl: 'https://walletconnect.com/privacy',
-  customWallets: ConstantsUtil.CustomWallets,
```

Comment:
Are we removing this because we will always display installed wallets ?

Reply:
Will add them back, removed them for testing purposes